src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs

```diff
@@ -41,12 +41,10 @@ public class InitialisationProcess : IInitialisationProcess
 
         public MutationTestInput Initialize(StrykerOptions options)
         {
-            _reporter.OnInitialisationStarted();
```

Comment:
I don't think the _reporter property is needed at all anymore here

Reply:
right